db/migrate/20240806061243_create_accounts_table.rb

```diff
@@ -0,0 +1,14 @@
+class CreateAccountsTable < ActiveRecord::Migration[7.1]
+  def change
+    create_table :accounts, id: false do |t|
+      t.string :account_number, primary_key: true
```

Comment:
never create a string PK, you can keep the database id as it is, and why use string for account_number?